src/shell.rs

```diff
@@ -93,6 +94,22 @@ mod tests {
             .stderr("");
     }
 
+    #[apply(opts)]
```

Comment:
Use `#[cfg(windows)]` for this and `#[cfg(unix)]` for the others.

Reply:
Done.